src/index.ts

```diff
@@ -88,3 +91,11 @@ export const plugin: PluginFunction = (schema, documents) => {
     content: content,
   }
 }
+
+export type Args = { schema: GraphQLSchema } & Partial<GraphQLArgs>
```

Comment:
I would omit `source` and `variableValues` from `GraphQLArgs` as they are always overridden by the test function :
```
request<${name}>({ ...graphqlArgs, source: ${name}Source, variableValues: variables })
```